core/modules/admin_bar/js/admin_bar.js

```diff
@@ -411,12 +411,14 @@ Backdrop.adminBar.behaviors.search = function (context, settings, $adminBar) {
     if (e.type === 'touchstart' && !$(this).hasClass('active-search-item')) {
       e.preventDefault();
     }
-    if (show) {
-      $adminBar.find('.active-search-item').removeClass('active-search-item');
-      $(this).addClass('active-search-item');
-    }
-    else {
+    $adminBar.find('.active-search-item').each(function () {
       $(this).removeClass('active-search-item');
+      if ($(this).is('[class=""]')) {
+        $(this).removeAttr('class');
```

Comment:
I guess what @quicksketch meant was this:

```suggestion
      $this.removeClass('active-search-item');
      if ($this.is('[class=""]')) {
        $this.removeAttr('class');
```

Reply:
Yes, however the code changed since then. You can't use that here since I added a new function, so now `$this != $(this)`.